### PR TITLE
Modified introduction section of the lesson.

### PR DESCRIPTION
**Pull request by rraffiu**
I have modified the introduction episode of the lesson. Some parts of the introduction were abstract and not easy to relate to. I have instead used a more relatable way of introducing shell. The episode is too long to be covered in five minutes time. Therefore, deleted some redundant information from the example.

### CODE REVIEW

File: _episodes/01-intro.md [hunk 1]
```diff
@@ -54,7 +47,7 @@ loops back and waits for you to enter another command.
 ### The Shell
 
 
-The Shell is a program which runs other programs rather than doing calculations itself.
+The Shell is a program which runs other programs rather than doing calculations itself. Those programs can be as complicated as a climate modeling software and as simple as a program that creates a new folder/director. The simple programs which are used to perform stand alone tasks are usually refered to as commands.
```
Comment (gdevenyi) [inline _episodes/01-intro.md]: director -> directory

File: _episodes/01-intro.md [hunk 2]
```diff
@@ -32,17 +32,10 @@ including through a keyboard and mouse, touch screen interfaces, or using speech
...
 done using traditional screens, mice, touchpads and keyboards.
 
-We are all familiar with **graphical user interfaces** (GUI): windows, icons and pointers.
-They are easy to learn and fantastic for simple tasks where a vocabulary consisting of
-"click" translates easily into "do the thing I want". But this magic relies on 
-wanting a simple set of things, and having programs that can do exactly those things.
+The **graphical user interface** (GUI) is the most widely used way to interact with personal computers. We give instructions (to run a program, to copy a file, to create a new folder/directory) with the convenience of a few mouse clicks. This way of interacting with a computer is intuitive and very easy to learn. But this way of giving instructions to a computer scales very poorly if we are to give large stream to instructions even if they are similar or identical. For example if we have to copy third line of each of a thousand text files stored in thousand different folders/directories and paste it into a single file line by line. Using the tradition GUI approach of clicks will take several hours to do this.
```
Comment (gcapes) [inline _episodes/01-intro.md]: large stream to instructions > large stream *of* instructions
if we have to copy third line > if we have to copy *the* third line

File: _episodes/01-intro.md [hunk 3]
```diff
@@ -32,17 +32,10 @@ including through a keyboard and mouse, touch screen interfaces, or using speech
...
+The **graphical user interface** (GUI) is the most widely used way to interact with personal computers. We give instructions (to run a program, to copy a file, to create a new folder/directory) with the convenience of a few mouse clicks. This way of interacting with a computer is intuitive and very easy to learn. But this way of giving instructions to a computer scales very poorly if we are to give large stream to instructions even if they are similar or identical. For example if we have to copy third line of each of a thousand text files stored in thousand different folders/directories and paste it into a single file line by line. Using the tradition GUI approach of clicks will take several hours to do this.  
 
-If you wish to do complex, purpose-specific things it helps to have a richer means
-of expressing your instructions to the computer. It doesn't need to be complicated or
-difficult, just a vocabulary of commands and a simple grammar for using them.
-
-This is what the shell provides - a simple language and a **command-line interface** 
-to use it through. 
+This is where we take advange of shell - a **command-line interface**
```
Comment (gcapes) [inline _episodes/01-intro.md]: take advantage of *the* shell

File: _episodes/01-intro.md [hunk 4]
```diff
@@ -32,17 +32,10 @@ including through a keyboard and mouse, touch screen interfaces, or using speech
...
 
-If you wish to do complex, purpose-specific things it helps to have a richer means
-of expressing your instructions to the computer. It doesn't need to be complicated or
-difficult, just a vocabulary of commands and a simple grammar for using them.
-
-This is what the shell provides - a simple language and a **command-line interface** 
-to use it through. 
+This is where we take advange of shell - a **command-line interface** 
+to make such repetitive tasks atumatic and fast. It can take a single instruction and repeat it over as it is or with some modification as many times as we want. The task in example obove can be accomplished in a few minutes at most.
```
[the remark this answers is not in the document]
Comment (gcapes) [reply _episodes/01-intro.md]: obove > above

[the remark this answers is not in the document]
Comment (gcapes) [reply _episodes/01-intro.md]: in example > in *the* example

**Comment by gcapes** [comment]
Should the lines be hard-wrapped? @gdevenyi

File: _episodes/01-intro.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (rraffiu) [reply _episodes/01-intro.md]: Thanks.

**Comment by gdevenyi** [comment]
@gcapes I don't think the style guide enforces anything right now.

I guess I line like breaks because it makes the diffs cleaner...

**Comment by gcapes** [comment]
@gdevenyi 
Found it! https://carpentries.github.io/lesson-example/04-formatting/#maximum-line-length

@rraffiu 
Could you please hard-wrap the lines, then I think this is good to merge!
Thanks for your efforts.

**Comment by gdevenyi** [comment]
Ah ha!. Yet another reason for me to figure out this travis CI autorun of our testing code...

**Comment by rraffiu** [comment]
Done.

**Comment by gcapes** [comment]
Great, thanks!

**Pull request merged**